2do_fe/src/redux/reducers/index.js

```diff
@@ -18,13 +18,20 @@ const taskReducer = (state = initialState, action) => {
 
     case types.GET_ALL_TASKS_SUCCESS:
     case types.DELETE_TASKS_SUCCESS:
-    case types.ADD_TASKS_SUCCESS:
       return {
         ...state,
         loading: false,
         tasks: payload,
       };
 
+    case types.ADD_TASKS_SUCCESS:
+      return {
+        ...state,
+        loading: false,
+        tasks: [...state.tasks,payload],
```

Comment:
sorted task before assign to tasks?

Reply:
Looking into it, would it be better to do that in Tasks List or nah?